### PR TITLE
Update Keras tool documentation

### PR DESCRIPTION
**Pull request by anuprulez**
The changes are the following:
- Updates Keras tool names
- Adds description

@qiagu @bgruening

### CODE REVIEW

File: galaxy_ml/keras_model_config.xml [hunk 1]
```diff
@@ -728,6 +728,23 @@
   <help>
       <![CDATA[
 **Help**
+
+**What it does**
+
+Creates a deep learning architecture using Keras deep learning library by adding different types of layers in a sequential manner. Different types of layers include Dense, Dropout, Convolutional, Activation and so on. These layers are added one after another which generates a deep architecture.
```
Comment (qiagu) [inline galaxy_ml/keras_model_config.xml]: Thanks a lot for adding 'help' text! This is great.
Have you tried making a `functional` model in the layers adding tool? The tool not only supports sequential model, but also functional model.

File: galaxy_ml/model_validation.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<tool id="sklearn_model_validation" name="Model Validation" version="@VERSION@">
-    <description>evaluates estimator performance by cross-validation</description>
+<tool id="sklearn_model_validation" name="Evaluate deep learning architecture" version="@VERSION@">
```
Comment (qiagu) [inline galaxy_ml/model_validation.xml]: This tool actually works for both sklearn and keras models.

File: galaxy_ml/keras_model_builder.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-<tool id="keras_model_builder" name="Keras Model Builder" version="@VERSION@">
-  <description>constructs a deep learning model with sciki-learn APIs</description>
+<tool id="keras_model_builder" name="Compile deep learning architecture" version="@VERSION@">
```
Comment (qiagu) [inline galaxy_ml/keras_model_builder.xml]: Personally, I like `Create deep learning model` better, or `Create deep learning estimator`.

**Comment by bgruening** [comment]
I like it!

File: galaxy_ml/model_validation.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
-<tool id="sklearn_model_validation" name="Model Validation" version="@VERSION@">
-    <description>evaluates estimator performance by cross-validation</description>
+<tool id="sklearn_model_validation" name="Evaluate scikit-learn and deep learning models" version="@VERSION@">
+    <description>performances by cross-validation</description>
     <macros>
```
Comment (qiagu) [inline galaxy_ml/model_validation.xml:3]: How about `Model Validation, evaluates estimator (including scikit-learn and deep learning) performance`?
I don't have strong preference though.
The same situation applies to searchcv tool, shared by sklearn, deep learning and other kinds of models.

**Comment by qiagu** [comment]
Thanks a lot. @anuprulez

**Pull request merged**